frontend/src/metabase/common/components/Table/useTableSorting.tsx

```diff
@@ -0,0 +1,65 @@
+import { useCallback, useMemo, useState } from "react";
+
+import { useLocale } from "metabase/common/hooks/use-locale/use-locale";
+
+import type { BaseRow } from "./types";
+
+const compareNumbers = (a: number, b: number) => (a < b ? -1 : 1);
```

Comment:
I think you actually want to return the difference in the numbers here, since the positive and negative bit is handled below on line 48